GDJS/Runtime/variablescontainer.ts

```diff
@@ -192,15 +192,58 @@ namespace gdjs {
       const variable = this._variables.get(name);
       return !!variable && !variable.isUndefinedInContainer();
     }
+
+    /**
+     * Check if a variable exists in the container.
+     * @param variable The variable
+     * @return true if the variable exists.
+     */
+    hasVariable(variable: gdjs.Variable): boolean {
+      const foundVariable = this._variablesArray.find((v) => v === variable);
+      return !!foundVariable && !foundVariable.isUndefinedInContainer();
+    }
+
+    getVariableNameInContainerByLoopingThroughAllVariables(
```

Comment:
I like this name. It's verbose but a clear sign for the caller that "huh, I should probably avoid doing this too much".